### ISSUE TITLE
FormatString is confused by other "format"s with EnforcedStyle: percent

### ISSUE THREAD

**Issue by aripollak** [rubocop/rubocop#1388]
I have this in .rubocop.yml:

``` yaml
FormatString:
  EnforcedStyle: percent
```

Rubocop finds false positives here:

```
 C: Favor String#% over format.
    controller.render format => body, status: status
                      ^^^^^^
 C: Favor String#% over format.    
    body = Rabl.render(resource_errors, 'errors/index', format: format)
                                                                ^^^^^^
```
